paywall/src/__tests__/hooks/browser/useListenForPostMessage.test.js

```diff
@@ -28,164 +32,145 @@ describe('useListenForPostMessage hook', () => {
     config = { isServer: false, isInIframe: true }
   })
 
-  it('sets up the event listener', () => {
-    expect.assertions(1)
-
-    rtl.testHook(() => useListenForPostMessage(fakeWindow), {
-      wrapper,
+  function MockListener({
+    /* eslint-disable react/prop-types */
```

Comment:
Any reason you need to disable these?

Reply:
Just seems weird to define prop types for a test helper, but I can do this if you'd like